### PR TITLE
Fix transformer loss

### PR DESCRIPTION
**Pull request by k-w-w**
Thanks for finding this @robieta

### CODE REVIEW

File: official/transformer/transformer_main.py [hunk 1]
```diff
@@ -83,7 +83,7 @@ def model_fn(features, labels, mode, params):
     # Calculate model loss.
     xentropy, weights = metrics.padded_cross_entropy_loss(
         logits, targets, params.label_smoothing, params.vocab_size)
-    loss = tf.reduce_sum(xentropy * weights) / tf.reduce_sum(weights)
+    loss = tf.reduce_sum(xentropy) / tf.reduce_sum(weights)
 
     # Save loss as named tensor that will be logged with the logging hook.
     tf.identity(loss, "cross_entropy")
```
Comment (robieta) [inline official/transformer/transformer_main.py]: As long as you're in here could you disambiguate xentropy and loss? this tf.identity name is quite confusing.

Comment (k-w-w) [reply official/transformer/transformer_main.py]: I've added comments describing xentropy and loss. Loss represents the overall cross entropy loss of the model, so the tf.identity name should be accurate.

**Comment by k-w-w** [comment]
Thanks!
